test/integration/force_torque.cc

```diff
@@ -0,0 +1,310 @@
+    _sensorSdf = sensor->Element();
+  }
+  else
+  {
+    _sensorSdf = nullptr;
+  }
+}
+
+/// \brief Test force torque sensor
+class ForceTorqueSensorTest
+    : public testing::TestWithParam<std::tuple<const char *, const char *>>
+{
+  // Documentation inherited
+  protected: void SetUp() override
+  {
+    ignition::common::Console::SetVerbosity(4);
+  }
+};
+
+/////////////////////////////////////////////////
+TEST_F(ForceTorqueSensorTest, CreateForceTorqueSensor)
+{
+  // Create SDF describing a force torque sensor
+  const std::string name = "TestForceTorqueSensor";
+  const std::string topic = "/ignition/sensors/test/force_torque";
+
+  const double updateRate = 30;
+  const bool alwaysOn = 1;
+  const bool visualize = 1;
+
+  // Create sensor SDF
+  ignition::math::Pose3d sensorPose(ignition::math::Vector3d(0.25, 0.0, 0.5),
+      ignition::math::Quaterniond::Identity);
+  sdf::ElementPtr forcetorqueSdf;
+  CreateForceTorqueToSdf(name, sensorPose, updateRate, topic, alwaysOn,
+                         visualize, "child", "child_to_parent", {}, {}, {}, {},
+                         forcetorqueSdf);
+
+  ASSERT_NE(nullptr, forcetorqueSdf);
+
+  // create the sensor using sensor factory
+  ignition::sensors::SensorFactory sf;
+  auto sensor =
+      sf.CreateSensor<ignition::sensors::ForceTorqueSensor>(forcetorqueSdf);
+  ASSERT_NE(nullptr, sensor);
+
+  EXPECT_EQ(name, sensor->Name());
+  EXPECT_EQ(topic, sensor->Topic());
+  EXPECT_DOUBLE_EQ(updateRate, sensor->UpdateRate());
+}
+
+/////////////////////////////////////////////////
+TEST_P(ForceTorqueSensorTest, SensorReadings)
+{
+  namespace math = ignition::math;
+
+  std::string frame, measureDirection;
+  std::tie(frame, measureDirection) = GetParam();
+
+  // Create SDF describing a a force torque sensor
+  const std::string name = "TestForceTorqueSensor";
+  const std::string topic = "/ignition/sensors/test/force_torque";
+  const double updateRate = 30;
+  const bool alwaysOn = 1;
+  const bool visualize = 1;
+
+  const math::Vector3d forceNoiseMean{0.1, 0.2, 0.3};
+  // Set the stddev to 0s so we can test the mean values
+  const math::Vector3d forceNoiseStddev{0.0, 0.0, 0.0};
+
+  const math::Vector3d torqueNoiseMean{0.5, 0.6, 0.7};
+  // Set the stddev to 0s so we can test the mean values
+  const math::Vector3d torqueNoiseStddev{0.0, 0.0, 0.0};
+
+  // Create sensor SDF
+  math::Pose3d sensorPose{math::Vector3d(0.25, 0.0, 0.5),
+                          math::Quaterniond::Identity};
+  sdf::ElementPtr forcetorqueSdf;
+
+  CreateForceTorqueToSdf(name, sensorPose, updateRate, topic, alwaysOn,
+                         visualize, frame, measureDirection, forceNoiseMean,
+                         forceNoiseStddev, torqueNoiseMean, torqueNoiseStddev,
+                         forcetorqueSdf);
+
+  ASSERT_NE(nullptr, forcetorqueSdf);
+
+  // create the sensor using sensor factory
+  // try creating without specifying the sensor type and then cast it
```

Comment:
I think this line doesn't apply anymore since `ign-sensors6`, I forgot to remove it from other tests

Reply:
Removed from relevant integration tests: fe1ff32